products/android.md

```diff
@@ -14,6 +14,11 @@ eolColumn: Security Support
 sortReleasesBy: 'release'
 releaseLabel: "Android __RELEASE_CYCLE__ '__CODENAME__'"
 releases:
+  - releaseCycle: "12L"
+    codename: Snow Cone 12.1
```

Comment:
@usta Looks good for me. I am not sure about the codename. Here https://en.wikipedia.org/wiki/Android_version_history it is named as `Snow Cone v2` but no source specified. Where have you `Snow Cone 12.1` from?

Reply:
It marked as 12.1 as version number i added both 12L and 12.1 to keep search engine optimization

Reply:
Sv2 only mentioned in android studio while selecting avd but i believe adding 12.1 to its name will bring more search results to us

Reply:
I have made it much better could you review again

Reply:
it's better now :+1: